smithy-typescript-ssdk-libs/server-common/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@aws-smithy/server-common",
+  "version": "1.0.0",
+  "description": "Base components for Smithy services",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "types": "./types/index.d.ts",
+  "scripts": {
+    "prepublishOnly": "yarn build:cjs && yarn build:es",
+    "pretest": "yarn build:cjs",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build": "yarn build:es && yarn build:cjs",
+    "postbuild": "downlevel-dts types types/ts3.4",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/awslabs/smithy-typescript.git",
+    "directory": "smithy-typescript-libs/smithy-server-common"
+  },
+  "author": "AWS Smithy Team",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@aws-sdk/smithy-client": "3.6.1",
+    "@aws-sdk/protocol-http": "^3.5.0",
+    "@aws-sdk/types": "^3.4.1",
+    "tslib": "^1.8.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.20",
+    "@types/node": "^12.7.5",
+    "downlevel-dts": "^0.7.0",
+    "jest": "^26.6.3",
+    "ts-jest": "^26.5.2",
+    "typescript": "^4.2.2"
+  },
+  "engines": {
+    "node": ">= 10.0.0"
```

Comment:
FYI, node 10 will be EOL at 7/30/2021. If it is suppose to be in lambda only, it's benefitial to just target at higher version to get better syntax.

Reply:
I think we should probably not support running services in the browser... but I could be wrong.

Reply:
Good call-out. I think we can reasonably target the latest version of node that Lambda supports (finalized at release time).